src/honey_badger/builder.rs

```diff
@@ -56,7 +66,7 @@ where
         };
         let step = if self.netinfo.is_validator() {
             // The first message in an epoch announces the epoch transition.
-            Target::All.message(Message::EpochStarted(0)).into()
+            Target::All.message(Message::EpochStarted(epoch)).into()
```

Comment:
I think we'd just need to remove this one line and rebase.
Then the DHB builder will need the same functionality.